org-cyf-itp/content/onboarding/sprints/3/prep/index.md

```diff
@@ -14,6 +14,9 @@ title="Accessibility audit"
 src="https://github.com/CodeYourFuture/Project-Record-A-Goose/"
 time=180
 [[blocks]]
+name="Learn about Interview Introductions"
```

Comment:
I generally put anything needed for the day-plan as the _first_ item of prep to make sure it definitely gets done - let's maybe move this to the top of the list?